tests/test_functions.py

```diff
@@ -209,7 +209,7 @@ def test_task_options(self, task_opts_params):
 
         schedule_time = datetime.fromisoformat(task['schedule_time'][:-1])
         delta = abs(schedule_time - _SCHEDULE_TIME)
-        assert delta <= timedelta(seconds=15)
+        assert delta <= timedelta(seconds=30)
```

Comment:
This unit test was failing during local test runs which current time delta. Removing this change from the commit since it's not in scope for this change.